README.rst

```diff
@@ -31,3 +31,17 @@ Development
 See https://github.com/laterpay/django-laterpay
 
 `Tested by Travis <https://travis-ci.org/laterpay/django-laterpay>`__
+
+Releasing checklist
+-------------------
+
+This repository uses `GitHub flow <https://guides.github.com/introduction/flow/index.html>`__.
+In order to release a new version please follow these steps:
+
+* Ensure ``CHANGELOG`` is representative
+* Determine next version number from the ``CHANGELOG`` (ensuring we follow `SemVer<http://semver.org/>`__)
```

Comment:
ITYM:

```
`SemVer <http://semver.org/>`_
```

(space and only one _)

I always wonder if there's a linter we can use for stuff like this, but afaict doc8's the only half-decent rST linter and I don't think it catches things like this